squid/datadog_checks/squid/squid.py

```diff
@@ -129,3 +135,21 @@ def parse_counter(self, line):
             return None, None
 
         return counter, value
+
+    def submit_version(self, headers):
+        server_version = headers.get("Server", "")
+
+        match = VERSION_REGEX.match(server_version)
+        if match is None:
+            self.log.debug("Squid version is Unknown")
+            return None
+
+        version = match.group(1)
+        self.log.debug("Squid version is %s", version)
```

Comment:
we can lose this log since we submit it on L152 or else it will log `Squid version is None`